azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/QueueDescription.java

```diff
@@ -71,6 +71,7 @@ public Duration getLockDuration()
     /**
      * Sets the duration for which a message will be locked while receiving through a PeekLock receiver.
      * Max value is 5 minutes.
+     * @param lockDuration - The lock duration for the message.
```

Comment:
>The lock duration for the message [](start = 29, length = 33)

Rephrase it or copy it from LockDuration documentation in .net framework client.